### PR TITLE
Develop/version 2.3 upgrade due to workflows

### PR DESCRIPTION
**Pull request by sinukaarel**
Removed autoresponder, api key for opt in.
Tokens auto generated.
Subdomain url parser.
Workflows api for autoresonders.
Version 1.1.0

### CODE REVIEW

File: opencart 2.3.0.0 - 2.3.0.2/upload/admin/controller/extension/module/smaily_for_opencart.php [hunk 1]
```diff
@@ -270,23 +264,23 @@ public function index() {
...
         }
 
-        // Customer sync autoresponder.
-        if (isset($this->request->post['smaily_for_opencart_autoresponder'])) {
-            $data['autoresponder'] = json_decode(html_entity_decode($this->request->post['smaily_for_opencart_autoresponder']), true);
-        } else {
-            $data['autoresponder'] = json_decode(html_entity_decode($this->config->get('smaily_for_opencart_autoresponder')), true);
-        }
         // Customer sync additional fields.
         if (isset($this->reqest->post['smaily_for_opencart_syncronize_additional'])) {
```
Comment (kaittodesk) [inline opencart 2.3.0.0 - 2.3.0.2/upload/admin/controller/extension/module/smaily_for_opencart.php]: Typo. Is `reqest`, should be `request`.

Comment (kaittodesk) [reply opencart 2.3.0.0 - 2.3.0.2/upload/admin/controller/extension/module/smaily_for_opencart.php]: Same typo seems to be in version 3.0+ package.

Comment (sinukaarel) [reply opencart 2.3.0.0 - 2.3.0.2/upload/admin/controller/extension/module/smaily_for_opencart.php]: Yup missed that. Only way when that shows effect is when users saves form and causes validation error. Then selected value would be reset. Will fix that for 3.0+ package separately.

Comment (kaittodesk) [reply opencart 2.3.0.0 - 2.3.0.2/upload/admin/controller/extension/module/smaily_for_opencart.php]: Excellent. Thank you!

**Pull request merged**